README.md

```diff
@@ -740,6 +740,9 @@ Services to securely store your Docker images.
 - [Scalable Microservices with Kubernetes](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615) Free Udacity course
 - [State of containers: a debate with CoreOS, VMware and Google](https://www.youtube.com/watch?v=IiITP3yIRd8) (27:38)
 - [SysAdminCasts: Introduction to Docker](https://sysadmincasts.com/episodes/31-introduction-to-docker) (15:49)
+- [Docker from scratch](https://www.youtube.com/playlist?list=PLLhEJK7fQIxD-btrjrqdEfQHbkZnQrmqE) on YouTube by Paris Nakita Kejser
```

Comment:
1. The entries should sorted alphabetically. 
2. The time of the video(s) should be added.

Reply:
Thanks for feedback, i have updated now